### PR TITLE
A more verbose explanation of alias and include keys of appearance selector properties

### PR DESCRIPTION
**Pull request by voger**
This PR is an attempt to make a more verbose explanation of the difference between the `include` and `alias` keys of a selector property. Also it replaces the example of `spinner/button` with that of a window so it is easier to show visually the results of the keys.

### CODE REVIEW

File: docs/desktop/gui/appearance.md [hunk 1]
```diff
@@ -242,93 +242,140 @@ runtime of the `style` method. This allows to use more complex statements to
...
+![Widget with alias and include](appearance/widget_with_alias_and_include.png)
+
 When `alias` and `include` are identically pointing to the same selector the
-result is identical to the real alias
+result is the same as the string alias.
+
+### Includes
+
+Includes are used to reuse the result of another key and merge it with the local
+data. Includes may also used standalone without the `style` key but this is
```
Comment (derrell) [inline docs/desktop/gui/appearance.md:338]: may also BE used standalone

**Review by derrell — CHANGES_REQUESTED** [review]
Much more clear. Thanks!

File: docs/desktop/gui/appearance.md (hunk 1 above)
Comment (voger) [reply docs/desktop/gui/appearance.md]: Fixed

**Review by derrell — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Comment by rommni** [comment]
Thank you, really much more clear, before today i was still using all of these things without totally understood how it works. I think I will have to clean my code a little now :)

**Comment by cboulanger** [comment]
4 approvals (2 are necessary for doc changes), merging. Thank you @voger ! It would be great if you could also "backport" the change to the V6 docs at https://github.com/qooxdoo/documentation/tree/gh-pages/6.0 - thank you!

**Comment by voger** [comment]
@cboulanger sure but how do I do that? Should I clone that repo too and add the changes there also?

**Comment by cboulanger** [comment]
Yes, that's the fastest way at the moment - unfortunately, we have no automatic "backport" mechanism in place which would copy over changes in the repository to the archived version in http://github.com/qooxdoo/documentation 's `gh-pages` branch. Thank you very much!

**Pull request merged**